src/gui/editors/PianoRoll.cpp

```diff
@@ -202,10 +202,10 @@ PianoRoll::PianoRoll() :
 	signalMapper = new QSignalMapper( this );
 	m_semiToneMarkerMenu = new QMenu( this );
 
-	QAction * markSemitoneAction = new QAction( tr("Mark/unmark current semitone"), this );
-	QAction* markScaleAction = new QAction( tr("Mark current scale"), this );
-	QAction* markChordAction = new QAction( tr("Mark current chord"), this );
-	QAction* unmarkAllAction = new QAction( tr("Unmark all"), this );
+	QAction *markSemitoneAction = new QAction( tr("Mark/unmark current semitone"), this );
+	QAction *markScaleAction = new QAction( tr("Mark current scale"), this );
+	QAction *markChordAction = new QAction( tr("Mark current chord"), this );
+	QAction *unmarkAllAction = new QAction( tr("Unmark all"), this );
```

Comment:
I was pretty much trying to make the pointer declarations consistent. I think I've seen all possible formats there. I would personally always put it on the right to avoid unclear declarations like
```cpp
char* a, b;
```